base/src/test/java/io/spine/io/GlobSpec.java

```diff
@@ -37,7 +37,7 @@
 import static com.google.common.truth.Truth.assertWithMessage;
 
 @DisplayName("`Glob` Java API should expose")
-class GlobTest {
+class GlobSpec {
```

Comment:
Same as above.

Reply:
Converted both to Kotlin. I see your point, but even in Java the `Spec` suffix makes sense because it makes tests a specification document, thus changing attitude of junior and mid-level developers to this code.